### ISSUE TITLE
[Suggestion] Improved galactic core

### ISSUE THREAD

**Issue by Magnymbus** [tsunamayo/Starship-EVO#3154]
Namely, brighter ambient light and slower lightcruise/smaller warp jumps due to increased gravity.

Perhaps a supermassive black hole at the very center... but you'd need a good light warping shader for that.

I got to a planet called Pitans, which is as close to the very center as possible. The space skybox is VERY bright, but the shadows are still just as dark as the starting planet.

**Comment by tsunamayo** [comment]
Yes I would like to do something specific for the core, I will see in time. thanks
